common/test/keyboards/issue6005/issue6005.keyboard_info

```diff
@@ -0,0 +1,7 @@
+{
+    "license": "mit",
+    "languages": [
+        
+    ],
+	  "description": "Vowel Hater generated from template"
```

Comment:
Ditto (actually we don't need the .keyboard_info file at all, as this is only for keyboards that will be deployed to keyman.com)